### PR TITLE
Support call to cancel all orders on a symbol

### PR DESCRIPTION
**Pull request by iterums**
Cancel all open orders on a symbol API call not supported #258

### CODE REVIEW

File: test/auth.js [hunk 1]
```diff
@@ -124,6 +124,14 @@ const main = () => {
     }
   })
 
+  test('[REST] cancelOpenOrders', async t => {
+    try {
+      await client.cancelOpenOrders({ symbol: 'ETHBTC'})
```
Comment (balthazar) [inline test/auth.js:129]: Can you add the missing space before the `}`?

Comment (iterums) [reply test/auth.js]: done

**Comment by balthazar** [comment]
Thanks!

**Comment by iterums** [comment]
Just noticed in the readme, the table has yet to be updated to be consistent with the other examples. I just copied the table from the Binance API and forgot to update it.

It should be:

| Param      | Type     | Required  | Description                               |
|------------|----------|-----------|-------------------------------------------|
| symbol     | String   | true      |                                           |
| recvWindow | Number   | false     | The value cannot be greater than `60000`  |

**Comment by balthazar** [comment]
Yeah can you make a new PR? I think recvWindow can also be omitted too as it's a global param

**Pull request merged**